themes/finna2/templates/Recommend/FinnaStaticHelp.phtml

```diff
@@ -0,0 +1,18 @@
+<!-- START of: finna - Recommend/FinnaStaticHelp.phtml -->
+<?php
+    $searchHelpPage = 'help-' . ('advanced' === $this->recommend->getSearchType() ? 'advanced' : 'basic');
```

Comment:
Palauttaako this->recommend->getSearchType nuo samat arvot?

Reply:
Melko varmasti kyllä, mutta tuo on siltä varalta, että olisi jotain muutakin, kun meillä ei ole helppisivuja millekään muulle.